tools/mvm.sh

```diff
@@ -421,7 +421,7 @@ fi
 # check if mesos source directory exists
 if [ ! -d "$MESOS_SOURCES" ]; then
   echo "Mesos sources not found. Cloning sources."
-  git clone https://git-wip-us.apache.org/repos/asf/mesos.git "$MESOS_SOURCES"
+  git clone https://github.com/apache/mesos.git "$MESOS_SOURCES"
```

Comment:
Looks like it should be https://gitbox.apache.org/repos/asf?p=mesos.git

Reply:
Updated according to http://mesos.apache.org/documentation/latest/building/. Took the version w/o special char like '?' to prevent any possible issues using it inside sh/cmd.